config/sync/core.extension.yml

```diff
@@ -5,6 +5,7 @@ module:
   admin_toolbar: 0
   admin_toolbar_links_access_filter: 0
   autocomplete_id: 0
+  better_exposed_filters: 0
```

Comment:
This module is used to make the "Include in homepage updates" a checkbox (as otherwise it would need to be a dropdown, which is a bit odd as it's just a boolean field).